idea_town/experiments/serializers.py

```diff
@@ -51,6 +52,9 @@ def get_installations_url(self, obj):
                        args=(obj.pk,))
         return request.build_absolute_uri(path)
 
+    def get_installation_count(self, obj):
```

Comment:
This would be better implemented as [a property](https://docs.python.org/3.5/library/functions.html#property) on the Experiment model itself, rather than just in the API serializer. When necessary, we can cache it there along with other model instance data.